editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -59,7 +62,7 @@ export function gridCellCoordinates(row: number, column: number): GridCellCoordi
 }
 
 export function gridCellCoordinatesToString(coords: GridCellCoordinates): string {
-  return `${coords.row}:${coords.column}`
```

Comment:
why this change?

Reply:
because there's no reason to have those serialized in two different ways (hyphen vs colon) and in my WIP code it was a meaningful difference – **however** I now realized it's actually never used anymore, so I removed it [`1ed278b` (#6046)](https://github.com/concrete-utopia/utopia/pull/6046/commits/1ed278b3e453c8bbe6f77587af683d779fdd1add)